docs/available-filters/record-encryption/aws-kms/setup.adoc

```diff
@@ -0,0 +1,108 @@
+:aws:  https://docs.aws.amazon.com/
+
+= AWS Key Management Service
+
+The filter is able to integrate with {aws}/kms/latest/developerguide/overview.html[AWS Key Management Service].
+
+Follow the instructions below to prepare your AWS account.  You'll need a privileged AWS user
```

Comment:
So, I've been using my root account for my AWS instance to perform the setup steps.  I've been running the commands in the CloudShell (a new party trick I learnt over the last 24hours).